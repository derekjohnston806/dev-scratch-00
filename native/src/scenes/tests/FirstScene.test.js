/**
    FirstScene.test.js

    @author:
      - Derek Johnston

    @desc:
      - Provide a test for the FirstScene component.
*/
import "react-native";
import React from "react";
import FirstScene from "../FirstScene.react";

import renderer from "react-test-renderer";

test("Should render itself correctly.", () => {
  const tree = renderer.create(<FirstScene />).toJSON();
  expect(tree).toMatchSnapshot();
});
