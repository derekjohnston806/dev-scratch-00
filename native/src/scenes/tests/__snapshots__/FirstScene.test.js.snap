exports[`test Should render itself correctly. 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#17B14B",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#E6E6E6",
        "fontSize": 36,
      }
    }>
    First Scene
  </Text>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        undefined,
      ]
    }
    testID={undefined}
    tvParallaxProperties={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#395E66",
          "fontSize": 24,
        }
      }>
      Go to Second Scene
    </Text>
  </View>
</View>
`;
